wd_helper.au3

```diff
@@ -652,27 +652,16 @@ EndFunc   ;==>_WD_FrameLeave
 ; Example .......: No
 ; ===============================================================================================================================
 Func _WD_HighlightElement($sSession, $sElement, $iMethod = Default)
```

Comment:
I know that this will be `Script Breaking Changed` but 

`_WD_HighlightElement` have exactly the same parameters like `_WD_HighlightElements` and IMO `_WD_HighlightElement` could be deleted.

For script writers it will be very easy to fix. just by changing `_WD_HighlightElement` into `_WD_HighlightElements` string in all scripts for example using `Grep4Win` or manually using SciTE like editors.

Reply:
That's correct. But I think the correct way to handle this is to mark the function as deprecated and completely remove it later on.

Reply:
> But I think the correct way to handle this is to mark the function as deprecated and completely remove it later on.

Good idea. Will you set a deletion deadline to 2023?

Reply:
Haven't decided when that will occur. However, I don't think it needs to be a year out.